bin/picca_Pk1D_cross_exposure.py

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env python
+"""Compute the individual cross-exposure 1D power spectra
+"""
+
+import sys, os, argparse, glob
+import fitsio
+import numpy as np
+from picca.pk1d.compute_pk1d import compute_pk_cross_exposure, Pk1D
+from picca.utils import userprint
+import multiprocessing as mp
+from functools import partial
+
+
+def treat_pk_file(out_dir, filename):
+    """
+    Takes a single file containing the FFT of delta for
+    multiple exposures and computes the cross-exposure power spectrum. The function
+    returns nothing, but writes to disk a new fits file with all the information needed
+    to compute Pk_cross_exposure. This is done by looping over each targetid and chunkid, 
+    and computing Pk_cross_exposure for each pair of exposures.
+    
+    Arguments
+    ---------
+    out_dir: string
+    The directory path where the cross-exposure will be written
+
+    filename: string
+    The file for which to compute cross-exposure
+
+    Return
+    ------
+    None
+    """
+    fft_delta_list = []
+    file_out = None
+    file_number = filename.split("-")[-1].split(".fits.gz")[0]
+    with fitsio.FITS(filename, "r") as hdus:
+        for _, hdu in enumerate(hdus[1:]):
+            fft_delta = Pk1D.from_fitsio(hdu)
+            if len(fft_delta.los_id.split("_")) < 2:
```

Comment:
Here CHUNKID comes from an older PR: it is number of a sub-forest after cutting in three sub-forests.
I think you are mistaking it with an exposure id. And yes, in that case this will be changed because the DLA masking is not done properly when I name the forests TARGETID_EXPOSUREID.
Yes the best solution is to create a new field which is EXPOSUREID, but it requires way more work since, for the tile format for example, there are some exposures that are located on different tiles. 
I used the TARGETID_EXPOSUREID as a provisory solution, I will change that

Reply:
ah, sorry that's right. I was so much into multi-exp thinking that I forgot about the chunking after...
I guess one could always use either EXPID (in case of using non-coadded spectra files as input) or TILEID (in case of using coadds from tile-based structures) as a descriminator and store them in a unified name to the outputs of delta_extraction, then read that unified name here...